cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -26,3 +26,12 @@ options:
     default: "stable"
     description: |
       Snap channel to install Kubernetes master services from
+  api-extra-args:
+    type: string
+    default: ""
+    description: |
+      Space separated list of flags and key=value pairs that will be passed as arguments to
+      kube-apiserver. For example value like this:
+        runtime-config=batch/v2alpha1=true profiling=true
```

Comment:
I disagree on this, +1 to the description as-is. When I look at `juju config myapp`, I want to know exactly what each option does.